src/main/java/cms/gov/madie/measure/services/TestCaseService.java

```diff
@@ -65,6 +61,7 @@ protected TestCase enrichNewTestCase(TestCase testCase, String username) {
     enrichedTestCase.setResourceUri(null);
     enrichedTestCase.setHapiOperationOutcome(null);
     enrichedTestCase.setValidResource(false);
+    enrichedTestCase.setPatientId(UUID.randomUUID());
```

Comment:
not part of your tickets. should we do similar while updating the test case as well to make sure the user doesn't override the existing patient Id?

Reply:
@adongare Thanks Ashok! I will do that with MAT-5907.